CWE/events/kenya.txt

```diff
@@ -8,7 +8,7 @@ country_event = {
 	fire_only_once = yes
 	
 	trigger = {
-		tag = ENG
+		OR = { tag = ENG tag = FBU }
```

Comment:
FBU must not be added as that is an AH situation where the empire is in a better situation to survive decolonisatio  and hence should not receive historical decolonisation events.

Reply:
I will remove it from the Tanganyka events too then

Reply:
Alright